hasher-matcher-actioner/Makefile

```diff
@@ -26,7 +25,9 @@ terraform/terraform.tfvars:
 	echo 'hma_lambda_docker_uri = "${DOCKER_URI}:${DOCKER_TAG}"' > terraform/terraform.tfvars
 
 terraform/backend.tf:
-	./scripts/write_backend_config.sh > terraform/backend.tf
+	@echo terraform state will be stored locally unless a backend.tf file is configured. 
+	@echo Create a backend.tf? to use custom s3 bucket run: export TF_STATE_S3_BUCKET=bucket_name [y/N]
+	@read ans; if [ $$ans = "y" ]; then ./scripts/write_backend_config.sh > terraform/backend.tf; fi
 
 dev_create_configs: terraform/terraform.tfvars terraform/backend.tf
```

Comment:
Due to how make works, it will keep prompting you every single time you run this if you say no. Is no considered a failure?

Reply:
yes, but after the first run of the command. I think it is more likely that `terraform apply` will be run instead. If they do use the make commands this will be a reminder that their state is not backed up anywhere and can be suppressed by adding:
```
terraform {
  backend "local" {
    path = "relative/path/to/terraform.tfstate"
  }
}
```
source: https://www.terraform.io/docs/language/settings/backends/local.html#example-configuration